### PR TITLE
Add PyCon Pune and PyConf Hyderabad

### PR DESCRIPTION
**Pull request by userimack**
**Details of the conference**:

- Website: Pycon Pune 
- Date: 08-11 Feb
- Venue: Pune 
- Description: PyCon Pune is a conference which includes first two days conference(08 - 09) and then two days dev spirint (10 - 11) to contribute to upstream projects.
-    -    -   -
- Website: PyConf Hyderabad 
- Date: 07-08 Oct
- Venue: IIIT Hyderabad, Hyderabad
- Description: First Python Conference in Hyderabad for the development of Python programming language.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ If you would like to be notified of future conferences that would be added to th
 | Conference | Date | Venue | Description |
 |------------|------|-------|-------------|
 | [Hosting and Domain Conference India](http://www.hdcon.org/) | 3 Feb | Hyderabad | Hosting and Domain Conference is the World's Top initiative to discuss on latest happenings in the field of web technologies. Hosting, Domain, Blogging and Web technologies are major discussions. |
+| [PyCon Pune](https://pune.pycon.org/2018/) | 08-11 Feb | Pune | PyCon Pune is a conference which includes first two days conference(08 - 09) and then two days dev spirint (10 - 11) to contribute to upstream projects. |
```
Comment (nikhita) [inline README.md]: Can you change the description to just `The community will meet for two days of talks and work on upstream projects in two days of dev sprint`?

**Comment by nikhita** [comment]
Also, please squash the commits after the change. :)

File: README.md (hunk 1 above)
Comment (userimack) [inline README.md]: hi, I am unable to squash the commit.

Comment (nikhita) [inline README.md]: That's ok - I'll squash using the Github UI. :smile: 
Just send in another commit with the change -- that would do.

Comment (userimack) [inline README.md]: thanks

Comment (userimack) [inline README.md]: using that command it is required to do a fast-forward push can I do that?

**Comment by nikhita** [comment]
Thanks a lot, @userimack! :)

File: README.md (hunk 1 above)
Comment (nikhita) [inline README.md]: I have merged the commits now. So you don't need to do anything right now.
But yes, whenever you squash commits, you will have to force push the change (and that's ok)!

Comment (userimack) [inline README.md]: i was facing that issue but since this is not my project so I thought of not doing a foce push. Thanks

Comment (nikhita) [inline README.md]: No problem! Even I was afraid of doing force pushes before. :)

**Pull request merged**